src/colors-variables.scss

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
Where were we storing colors before?

Reply:
There are currently 2 TypeScript files that store colors:

1. **/src/colors.tsx** (which contains the main Firebase colors and the colors are shared to scss using **ThemeProvider**):

![image](https://user-images.githubusercontent.com/34299428/104385247-81b9eb80-54f8-11eb-8296-e4565716a2d0.png)

2. **/src/themes.tsx** (which contains several themes objects that are also shared to scss using **ThemeProvider**):

![image](https://user-images.githubusercontent.com/34299428/104385362-baf25b80-54f8-11eb-801f-0fa7ef9e4971.png)

The alternative of not creating this scss file with colors variables will be to add them to the **/src/colors.tsx** file and share the colors from TypeScript to scss using **ThemeProvider**. I can do that and delete this scss file if it helps, there is just one exception where I couldn't find a way of sharing colors from TypeScript to scss, but I will discuss that case later with you folks (it is not included on this PR)

Reply:
Please revert this file and use ThemeProvider instead

Reply:
Done! ThemeProvider is indeed a lot better than creating another colors file, my bad 😅